src/Components/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect } from "react";
+import { useQuery, gql } from '@apollo/client'
+import { NavLink, useLocation, Routes, Route } from "react-router-dom";
+import DashboardMenu from '../DashboardMenu/DashboardMenu'
+import MyBooks from '../MyBooks/MyBooks'
+
+interface Location {
+  pathname: string,
+  search: string,
+  hash: string,
+  state: null,
+  key: string
+}
+
+
+const Dashboard = () => {
+  let location: Location = useLocation()
```

Comment:
Curious as to what you're using location for

Reply:
Yep! Remanent of past iteration of Dashboard.... removing